packages/forms/frontend/sirius-components-widget-reference/src/components/ValuedReferenceAutocomplete.tsx

```diff
@@ -115,7 +186,11 @@ export const ValuedReferenceAutocomplete = ({
       multiple
       filterSelectedOptions
       disabled={readOnly || widget.readOnly}
-      options={widget.referenceOptions}
+      open={open}
+      onOpen={() => setOpen(true)}
```

Comment:
Why aren"t you calling "getReferenceValueOptions" in this function? You are setting an "open" state which will trigger a new rendering which will change the value of loading which will be detected by useEffect to perform the call to the function.

Reply:
I need a state `open` in the case where there is no option. I need to know that the component is opened and I can't only check the result of `getReferenceValueOptions`.